pre_commit_mirror_maker/rust/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "__fake_crate"
+version = "0.0.0"
+
+[[bin]]
+name = "__fake_cmd"
+path = "main.rs"
```

Comment:
It turns out you can't really make a noop `Cargo.toml` file, and since we're installing with `--bins`, you can't really even make a library-only crate (so there would be no binary); cargo is "smart" enough to complain about that.

So this builds a bogus executable which does nothing. Maybe okay?

An alternative could be to make pre-commit's Rust support allow repos with no `Cargo.toml` (and only use `additional_dependencies` in that case), but I'm worried about it masking bugs.

Reply:
this is probably fine, we have to do the same dance for `go` as well